### ISSUE TITLE
a11y: Fix keyboard navigation and focus display for banner control close

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12784]
a11y Issue: 180

The banner control closebutton (see below) does not show keyboard focus and does not response to enter/space to activate.

![image](https://github.com/user-attachments/assets/96ec0d5e-e741-40ef-9b8b-5daae34ae45c)

Screenshot shows pod security admission screen which should be checked - but fix in the Banner component, so that all uses will be fixed.

**Comment by aalves08** [comment]
Fixed by PR https://github.com/rancher/dashboard/pull/12966. Moving it to QA testing

https://github.com/user-attachments/assets/d4926853-625c-48d4-9e63-17d1379e7ef5

FYI @rak-phillip @nwmac

**Comment by IsaSih** [comment]
Tested warning and error banners. Tests pass in 
Rancher | v2.11-3d729db8a848fb2523bbc2f3f21aa5367dcdcc69-head
Dashboard | master df18acc
Rancher CLI | v2.11.0-rc.1

https://github.com/user-attachments/assets/6611e93c-15b5-4e7c-8766-dc95763e9d05
